subrpcserver_config.go

```diff
@@ -107,6 +113,7 @@ func (s *subRPCServerConfigs) PopulateDependencies(cfg *Config,
 	graphDB *channeldb.ChannelGraph,
 	chanStateDB *channeldb.ChannelStateDB,
 	sweeper *sweep.UtxoSweeper,
+	neutrinoCS *neutrino.ChainService,
```

Comment:
I think is is actually accessible w/ the chain control pointer above?

Reply:
It is, that makes it easier. Thanks.